gradle.properties

```diff
@@ -14,3 +14,7 @@ version = 0.4.0-SNAPSHOT
 #sonatype credentials
 SONATYPE_USER = admin
 SONATYPE_PASSWORD = password
+
+#jdkVersion is used to specify the version of JDK to build and test against Gravitino, current
+# supported version is 8, 11, and 17.
+jdkVersion = 8
```

Comment:
What if the value of `jdkVersion` differs from that of the JDK installed actually?

Reply:
It is the same as before, if the specified `jdkVersion` is not installed locally, it will download automatically by gradle.

Reply:
I mean if the value of `jdkVersion` is 8 and the JDK you installed is JDK11 or JDK17, so the final behavior is?

Reply:
it will build, test against jdk8, no matter which version you installed in your environment.